kernel/packages/unity-interface/portableExperiencesUtils.ts

```diff
@@ -0,0 +1,162 @@
+const STATIC_PORTABLE_SCENES_S3_BUCKET_URL = 'https://static-pe.decentraland.io'
+export type PortableExperienceUrn = string
+
+declare var window: any
+window['spawnPortableExperienceScene'] = spawnPortableExperienceScene
+window['killPortableExperienceScene'] = killPortableExperienceScene
+
+export type PortableExperienceIdentifier = string
+export type PortableExperienceHandle = {
+  pid: PortableExperienceIdentifier
+  parentCid: string
+}
+
+let currentPortableExperiences: Map<string, string> = new Map()
+
+export function getPortableExperience(pid: string): PortableExperienceHandle | undefined {
+  if (currentPortableExperiences.has(pid)) {
+    return { pid: pid, parentCid: currentPortableExperiences.get(pid)! }
+  } else {
+    return undefined
+  }
+}
+
+export async function spawnPortableExperienceScene(
+  portableExperienceUrn: PortableExperienceUrn,
+  parentCid: string
+): Promise<PortableExperienceHandle> {
+  const parsedUrn: DecentralandAssetIdentifier | null = await parseUrn(portableExperienceUrn)
+
+  if (!parsedUrn || !isPortableExperience(parsedUrn)) {
+    throw new Error(`Could not parse portable experience from urn: ${portableExperienceUrn}`)
+  }
+
+  /* tslint:disable-next-line */
+  const scene = new UnityPortableExperienceScene(await getPortableExperienceFromS3Bucket(parsedUrn as OffChainAsset))
+  loadParcelScene(scene, undefined, true)
+  const parcelSceneId = getParcelSceneID(scene)
+  unityInterface.CreateUIScene({
+    id: parcelSceneId,
+    name: scene.data.name,
+    baseUrl: scene.data.baseUrl,
+    contents: scene.data.data.contents,
+    icon: scene.data.data.icon,
+    isPortableExperience: true
+  })
+
+  currentPortableExperiences.set(parcelSceneId, parentCid)
+
+  return { pid: parcelSceneId, parentCid: parentCid }
+}
+
+function isPortableExperience(dclId: DecentralandAssetIdentifier): dclId is OffChainAsset {
+  if (dclId) {
+    /* tslint:disable-next-line */
+    return !!(dclId as OffChainAsset).registry && (dclId as OffChainAsset).registry === 'static-portable-experiences'
+  }
+  return false
+}
+
+export async function killPortableExperienceScene(portableExperienceUrn: PortableExperienceUrn): Promise<boolean> {
+  const parsedUrn: DecentralandAssetIdentifier | null = await parseUrn(portableExperienceUrn)
+  if (!parsedUrn || !isPortableExperience(parsedUrn)) {
+    throw new Error(`Could not parse portable experience from urn: ${portableExperienceUrn}`)
+  }
+  const sceneId: string = parsedUrn.id
+  const peWorker = getSceneWorkerBySceneID(sceneId)
+  if (peWorker) {
+    forceStopParcelSceneWorker(peWorker)
+    currentPortableExperiences.delete(sceneId)
+    unityInterface.UnloadScene(sceneId)
+    return true
+  } else {
+    return false
+  }
+}
+
+export async function getPortableExperienceFromS3Bucket(pe: OffChainAsset) {
+  const peId: string = pe.id
+  const baseUrl: string = `${STATIC_PORTABLE_SCENES_S3_BUCKET_URL}/${peId}/`
+
+  const mappingsFetch = await fetch(`${baseUrl}mappings`)
```

Comment:
We have a function to perform the URN to URL resolution https://github.com/decentraland/urn-resolver/blob/main/test/content-url.spec.ts#L12-L15

Using that function is good because it will enable us to change and extend the resolution of portable experiences beyond the static endpoint and removes hardcoded dependencies in this project (STATIC_PORTABLE_SCENES_S3_BUCKET_URL)

```suggestion
// import {resolveContentUrl} from '@dcl/urn-resolver'
const mappingsUrl = await resolveContentUrl(pe)
const baseUrl: string = (new URL('..', mappingsUrl)).toString()
const mappingsFetch = await resolveContentUrl(mappingsUrl)
```